ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -590,8 +590,10 @@ public static void update_validator_registry(BeaconState state) {
                   .getExit_epoch()
```

Comment:
On line 588 (i don't think you can see it here):

`List<UnsignedLong> balances = state.getValidator_balances();`

it is unused.  do you mind deleting it for me?

Reply:
Oh yeah I skipped that.

Reply:
Just fixed it